changelog/README.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 2nd November 2022
+
+* Extended [Outlet item](../operations/outletitems.md) with `ExternalIdentifier`.
```

Comment:
I think the reference should be [Outlet item](../operations/outletitems.md#outlet-item)